packages/repack/src/webpack/utils/adaptFilenameToPlatform.ts

```diff
@@ -0,0 +1,10 @@
+import os from 'os';
+
+const isWindows = os.platform() === 'win32';
+
+export const adaptFilenameToPlatform = (filename: string) => {
+  if (isWindows) {
+    return filename.replace(/\\/g, '/');
```

Comment:
not sure if we use it already, but if we de, we could use `slash` library to do so